00-Raster-Structure.Rmd

```diff
@@ -64,13 +64,11 @@ preferably RStudio, loaded on your computer.
 ### Download Data
 {% include/dataSubsets/_data_Airborne-Remote-Sensing.html %}
 
-****
+## Things To Know To Complete This Lesson
```

Comment:
It is not inline with the current model but I think it is a good suggestion to think about.  I'm going to add it to the Issue (https://github.com/NEONInc/NEON-Data-Skills-Development/issues/3) in the Data-Skills-Dev repo that is also about grey box styling.  For now I'd not include this suggestion but we will revisit it when we can implement it across the site.